src/doc/rustdoc/src/documentation-tests.md

```diff
@@ -31,6 +31,17 @@ let x = 5;
 
 There's some subtlety though! Read on for more details.
 
+## Assertions-dedicated macros
+
+Of course, `assert!`, `assert_eq!` and `assert_ne` can be used into your examples as in any Rust source code.
```

Comment:
Hmm, i'm not a fan of this phrasing. How about something like this?

> ## Passing or failing a doctest
>
> Like regular unit tests, regular doctests are considered to "pass" if they compile and run without panicking. So if you want to demonstrate that some computation gives a certain result, the `assert!` family of macros works the same as other Rust code:
>
> ```rust
> let foo = "foo";
>
> assert_eq!(foo, "foo");
> ```
>
> This way, if the computation ever returns something different, the code panics and the doctest fails.

-----

Also, be aware that we like to insert hard line breaks into the docs to keep a single line from running longer than 100 characters. You can use the other sections in this file as a guideline.

Reply:
I'll replace the changes.

> Also, be aware that we like to insert hard line breaks into the docs to keep a single line from running longer than 100 characters. You can use the other sections in this file as a guideline.

Understood.